shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle_disposition.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
Please use Copyright 2013 here, I know it's strange but that's how we avoid adding a new license file and all that stuff

Reply:
done.